tests/utils/mod.rs

```diff
@@ -12,6 +12,8 @@ pub const BUYER: [u64; 2] = [12, 34];
 pub const SELLER: [u64; 2] = [56, 78];
 pub const AMOUNT: [u128; 2] = [12345, 54321];
 pub const WALLET: [u128; 2] = [0, 1];
+pub const AMOUNT_REMAINDER: u128 = 20000;
+pub const NONEXISTEND_WALLET: u128 = 999999;
```

Comment:
Non existent or not existing
```suggestion
pub const NONEXISTENT_WALLET: u128 = 999999;
```